services/webdav/pkg/service/v0/service.go

```diff
@@ -346,7 +346,11 @@ func (g Webdav) Thumbnail(w http.ResponseWriter, r *http.Request) {
 		switch e.Code {
 		case http.StatusNotFound:
 			// StatusNotFound is expected for unsupported files
-			renderError(w, r, errNotFound(notFoundMsg(tr.Filename)))
+			message := notFoundMsg(tr.Filename)
+			if strings.Contains(e.Detail, "could not find space") {
```

Comment:
NOTE: this only works in `master`. But in `stable-3.0`, we don't get error message like `could not find space`. I couldn't find which commit had this change. So, the fix won't work for stable-3.0.

CC @2403905

Reply:
Maybe in reva